server/src/changedConfigs.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Can we make a change to this based off of Mike's comments?

Let's change to the below format:

`[
  {
    "type": "new",
    "name": "xyz",
    "readme": "readme.md"
  },
  {
    "type": "changed",
    "oldName": "abc",
    "newName": "xyz",
    "readme": "readme.md"
  }
]`

Sorry for the last-minute ask.